packages/orbit-components/src/InputField/__tests__/index.test.js

```diff
@@ -62,7 +62,7 @@ describe("InputField", () => {
     expect(screen.getByTestId("test")).toBeInTheDocument();
     expect(screen.getByTestId("prefix")).toBeInTheDocument();
     expect(screen.getByTestId("suffix")).toBeInTheDocument();
-    fireEvent.focus(input);
+    userEvent.tab(input);
```

Comment:
[`userEvent.tab`](https://testing-library.com/docs/ecosystem-user-event/#tabshift-focustrap) is meant to replicate the tabbing that user does and doesn't take an element as an argument, it takes different optional arguments. Flow was right. 😅  Remove other instances of this, too.
```suggestion
    userEvent.tab();
```